onnx/defs/nn/defs.cc

```diff
@@ -369,6 +372,52 @@ Output case #2: Y (test mode)
     .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
         "Constrain input and output types to float tensors.");
 
+OPERATOR_SCHEMA(InstanceNormalization)
+    .NumInputs(3)
+    .NumOutputs(1, 3)
+    .AllowConsumed({{0, 0}})
+    .SetDoc(R"DOC(
+Carries out instance normalization as described in the paper
```

Comment:
would you be kind to also include math equation instead of just referencing the paper? (I know Caffe2 docs are guilty of that but let's be better)